src/BizHawk.Client.EmuHawk/tools/Watch/RamWatch.Designer.cs

```diff
@@ -659,6 +663,24 @@ private void InitializeComponent()
             this.WatchListView.KeyDown += new System.Windows.Forms.KeyEventHandler(this.WatchListView_KeyDown);
             this.WatchListView.MouseDoubleClick += new System.Windows.Forms.MouseEventHandler(this.WatchListView_MouseDoubleClick);
             // 
+            // DoubleClickActionSubMenu
+            // 
+            this.DoubleClickActionSubMenu.DropDownItems.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.DoubleClickToEditMenuItem,
+            this.DoubleClickToPokeMenuItem});
+            this.DoubleClickActionSubMenu.Text = "Double-Click Action";

```

Comment:
```suggestion
            this.DoubleClickActionSubMenu.Text = "On Double-Clicking a Watch:";
```
Though if Enter has the same effect, I'm not sure...

Reply:
I figured keyboard navigation should also have access to that feature, but that most users wouldn't care or would figure it out. Explicitly referring to double-clicking seemed more intuitive overall than trying to find an all-encompassing term, and it didn't seem worth it to over-complicate the menu or have a super long description somewhere